legend-engine-testable/src/test/java/org/finos/legend/engine/testable/assertion/TestTestAssertionEvaluator.java

```diff
@@ -152,4 +154,22 @@ public void testEqualToJsonAssertionWithDecimalPrecision()
         Assert.assertTrue(assertionStatus instanceof AssertPass);
         Assert.assertEquals("assert1", assertionStatus.id);
     }
+
+    @Test
+    public void testEqualToAssertionWithFloatPrecision() throws IOException
+    {
+        ConstantResult constantResult = new ConstantResult("{\"some\": 2.0}");
+        ExternalFormatData data = new ExternalFormatData();
+        data.contentType = "application/json";
+        EqualToJson equalToJson = new EqualToJson();
+        equalToJson.id = "assert1";
+        equalToJson.expected = data;
+        data.data = "{\"some\":2.0}";
```

Comment:
Let's please add more scenarios with passes and failures, like (but not limited to)
- 2000000000000000000
- 2.000000000000000000
- 2.000000000000000001
- 2.0
- 20